lib/tests/streamlit/elements/date_input_test.py

```diff
@@ -254,3 +265,28 @@ def test_invalid_date_format_values(self, format: str):
         with self.assertRaises(StreamlitAPIException) as ex:
             st.date_input("the label", format=format)
         self.assertTrue(str(ex.exception).startswith("The provided format"))
+
+
+class DateInputInteractiveTest(InteractiveScriptTests):
+    def test_date_input_interaction(self):
+        """Test interactions with an empty date_input widget."""
+        script = self.script_from_string(
+            """
+        import streamlit as st
+
+        st.date_input("the label", value=None)
+        """
+        )
+        sr = script.run()
+        date_input = sr.date_input[0]
+        assert date_input.value is None
+
+        # Set the value to 10
```

Comment:
Nit: Set it to 10?

Reply:
Whoops, this is from number input copy/paste...